src/engine/sparqlExpressions/PrefilterExpressionIndex.cpp

```diff
@@ -456,13 +488,44 @@ template class LogicalExpression<LogicalOperator::OR>;
 
 namespace detail {
 
+//______________________________________________________________________________
+template <CompOp comparison>
+static std::unique_ptr<PrefilterExpression> makeMirroredExpression(
+    const IdOrLocalVocabEntry& referenceValue) {
+  using enum CompOp;
+  switch (comparison) {
```

Comment:
I think this can be more concise and use the `constexprnemss` of the comparison:

```
using P = std::pair<CompOp>;
constexpr static std::array map {P{LT, GT}, P{LE, GE}, ...};
constexpr auto getMirroredComparison = [](Comp op c) {
  for (const auto& [key, value] : map) {
    if (key == c) { return value;}
}
// Throw some exception here.
};
return std::make_unique<RelationalExpression<getMirroredComparison(comparison)>(referenceValue);
```

Or even simpler: I think I have a `ConstexprMap` tyep defined in some util header, that does the iteration for you, s.t. you don't even need the lambda.
With this, also codecov will be happier.